students/dae-hwan/user/views.py

```diff
@@ -0,0 +1,50 @@
+import json 
+
+from django.http  import JsonResponse
+from django.views import View
+
+from user.models  import User
+
+class SignUpView(View):
+    def post(self, request):
+        data                = json.loads(request.body)
+        email               = data['email']
+        password            = data['password']
+        nick_name           = data['nick_name']
+        phone_number        = data['phone_number']
+        email_validation    = '^[a-z0-9,_-.]+@[a-z0-9,_-]+\.[a-z0-9,_-.]+$'
+        password_validation = '.{8,}'
+
+        # 정보가 잘 입력되었을 때
+        try: 
+            # 이메일에 관한 에러
+            if '@' not in email or '.' not in email:
+                return JsonResponse({'message': 'email_validation'}, status = 400)
```

Comment:
```suggestion
                return JsonResponse({'message': 'invalid email'}, status = 400)
```
위와 같은 메시지가 조금 더 어울릴 것 같아요. 그리고 메시지 형식은 통일하는 것이 좋습니다. 이 부분도 프론트랑 약속해야 하는 부분이기 때문입니다. 대문자로만 하겠다 또는 소문자에 '_'을 사용하겠다 등으로요
코드에 있는 에러 메시지들 한 번씩 다시 확인해 주세요.